Dockerfile

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Unnecesary empty line.

Reply:
Also, this image basically does nothing. Is this expected (temporarily)?

Reply:
This is pretty much final version of this image. Everything required for the actual app will be inside the app repo. This is only bnanse for setting envs and adding necessary chromium bloatware for puppeteer.

Reply:
It is. If I understood correctly, this is just some kind of template containing what is needed for pdf-renderer package to work. Does this change anything about parametrization?